tests/integration/test_identify_service.py

```diff
@@ -288,8 +288,50 @@ def test_identify_with_geojson_returned_geometry(self):
         resp_2 = self.testapp.get('/rest/services/ech/MapServer/identify', params=params, headers=accept_headers, status=200)
         self.assertEqual(resp_2.content_type, 'application/json')
         self.assertGeojsonFeature(resp_2.json['results'][0], 2056)
+        self.assertEqual(len(resp_2.json['results']), len(resp_1.json['results']))
+        self.assertEqual(resp_1.json['results'][0]['id'], resp_2.json['results'][0]['id'])
+
+    def test_identify_with_geojson_returned_geometry_various_sr(self):
+        params = {'geometry': '600000,200000,601000,201000',
+                  'geometryType': 'esriGeometryEnvelope',
+                  'imageDisplay': '500,600,96',
+                  'mapExtent': '548945.5,147956,549402,148103.5',
+                  'tolerance': '1',
+                  'layers': 'all:ch.bav.haltestellen-oev',
+                  'geometryFormat': 'geojson',
+                  'sr': '21781'}
+
+        # LV03
+        resp_1 = self.testapp.get('/rest/services/ech/MapServer/identify', params=params, headers=accept_headers, status=200)
+        self.assertEqual(resp_1.content_type, 'application/json')
+        self.assertGeojsonFeature(resp_1.json['results'][0], 21781)
+        self.assertIn(resp_1.json['results'][0]['geometry']['type'], ['MultiPoint', 'GeometryCollection'])
+        resp1_ids = [d['id'] for d in resp_1.json['results']]
 
+        # LV95
+        params['sr'] = '2056'
+        params['geometry'] = shift_to_lv95(params['geometry'])
+        params['mapExtent'] = shift_to_lv95(params['mapExtent'])
+        resp_2 = self.testapp.get('/rest/services/ech/MapServer/identify', params=params, headers=accept_headers, status=200)
+        self.assertEqual(resp_2.content_type, 'application/json')
+        self.assertGeojsonFeature(resp_2.json['results'][0], 2056)
+        self.assertEqual(len(resp_2.json['results']), len(resp_1.json['results']))
         self.assertEqual(resp_1.json['results'][0]['id'], resp_2.json['results'][0]['id'])
+        resp2_ids = [d['id'] for d in resp_2.json['results']]
+        self.assertEqual(resp1_ids, resp2_ids)
+
+        # Web Mercator
+        # TODO requess is OK, geometries are not reprojected
```

Comment:
That is the main thing we need.

Reply:
they do get reprojected:
http://mf-chsdi3.dev.bgdi.ch/webmerc/rest/services/api/MapServer/ch.bafu.nabelstationen/RIG?sr=3857
vs
http://mf-chsdi3.dev.bgdi.ch/webmerc/rest/services/api/MapServer/ch.bafu.nabelstationen/RIG